packages/block-library/src/post-terms/hooks.js

```diff
@@ -16,9 +16,11 @@ export default function enhanceVariations( settings, name ) {
 	}
 	const variations = settings.variations.map( ( variation ) => ( {
 		...variation,
-		...( variationIconMap[ variation.name ] && {
-			icon: variationIconMap[ variation.name ],
-		} ),
+		...{
+			icon: variationIconMap[ variation.name ]
+				? variationIconMap[ variation.name ]
+				: postCategories,
```

Comment:
```suggestion
			icon: variationIconMap[ variation.name ] ?? postCategories,
```

I think this one is simpler.

Reply:
Thank you for the review! I think this one is simpler and better.